src/v1/controllers/VerifyPIVController.js

```diff
@@ -27,6 +27,7 @@ export default FormController.extend({
       const data = {
         fromURI: this.settings.get('relayState'),
         isCustomDomain: pivConfig.isCustomDomain,
```

Comment:
Maybe we should remove the reference to `pivConfig.isCustomDomain`

Reply:
Wouldn't this be redundant, if you're supplying a custom domain value it can just infer whether to use custom domain or not?

Reply:
Yeah it is redundant on purpose to make it both backward and forward compatible.